src/containers/SnippetArea.jsx

```diff
@@ -145,20 +146,20 @@ export class SnippetArea extends React.Component {
     }
      // Update a pre-existing snippet
     if (id) {
-      return dispatch(saveExisting())
+      dispatch(saveExisting())
```

Comment:
Since you removed the `return`s, the logic is no longer halted at those points. So I believe you will need to either add them back or else wrap wrap the logic starting on line `157` in an `else`.